UPGRADING.md

```diff
@@ -1,4 +1,116 @@
-# Guide to upgrading from url 0.x to 1.x
+# Upgrade guide
+
+This guide contains steps for upgrading crates in this project between major
+versions. Only the most common issues are covered here. For full details of
+changes, see the [changelog](CHANGELOG.md).
```

Comment:
Since there is no `CHANGELOG.md` let’s remove these two sentences.

Reply:
Done.